services.py

```diff
@@ -3,7 +3,7 @@
 
 async def check_lnaddress_format(username: str) -> bool:
     # check username complies with lnaddress specification
-    if not re.match("^[a-z0-9-_.]{3,15}$", username):
+    if not re.match("^[a-z0-9-_.]{3,210}$", username):
         assert False, "Only letters a-z0-9-_. allowed, min 3 and max 15 characters!"
```

Comment:
also change it for the assert description

Reply:
Does it hurt to allow more chars?

Reply:
Do you have a usecase for this?

Reply:
Well, no 210 is just a nice number. And generally I think artificially tight limits are bad. Can’t really see the downside of allowing more.

Reply:
> Can’t really see the downside of allowing more.

More dense QR codes, inability to have email with the same prefix, ...

Reply:
and the whole point of a lnaddress or email, that its somewhat rememberable and humanreadable :D, else you could just use LNURL instead of a lnaddress.